From 333046e691a293b8ddfe0e0341a9f5b5de515f92 Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date: Tue, 3 Sep 2013 10:16:19 -0400
Subject: [PATCH 536/625] statedump: remove KERN_DEBUG messages

They appear to be shown by default in dmesg of many distributions, and
they don't add much value (noisy).

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Signed-off-by: Jianchuan Wang <jianchuan.wang@windriver.com>
---
 drivers/staging/lttng2/lttng-statedump-impl.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/staging/lttng2/lttng-statedump-impl.c b/drivers/staging/lttng2/lttng-statedump-impl.c
index bf9457b..aecf521 100644
--- a/drivers/staging/lttng2/lttng-statedump-impl.c
+++ b/drivers/staging/lttng2/lttng-statedump-impl.c
@@ -381,7 +381,6 @@ int do_lttng_statedump(struct lttng_session *session)
 {
 	int cpu;
 
-	printk(KERN_DEBUG "LTT state dump thread start\n");
 	trace_lttng_statedump_start(session);
 	lttng_enumerate_process_states(session);
 	lttng_enumerate_file_descriptors(session);
@@ -409,7 +408,6 @@ int do_lttng_statedump(struct lttng_session *session)
 	__wait_event(statedump_wq, (atomic_read(&kernel_threads_to_run) == 0));
 	put_online_cpus();
 	/* Our work is done */
-	printk(KERN_DEBUG "LTT state dump end\n");
 	trace_lttng_statedump_end(session);
 	return 0;
 }
@@ -419,7 +417,6 @@ int do_lttng_statedump(struct lttng_session *session)
  */
 int lttng_statedump_start(struct lttng_session *session)
 {
-	printk(KERN_DEBUG "LTTng: state dump begin\n");
 	return do_lttng_statedump(session);
 }
 EXPORT_SYMBOL_GPL(lttng_statedump_start);
-- 
2.0.2

